flink-streaming-connectors/flink-connector-kafka-base/src/test/java/org/apache/flink/streaming/connectors/kafka/testutils/MockRuntimeContext.java

```diff
@@ -161,6 +163,11 @@ public Histogram getHistogram(String name) {
 	}
 
 	@Override
+	public MetricGroup getMetricGroup() {
+		return new UnregisteredTaskMetricsGroup.DummyIOMetricGroup();
```

Comment:
You don't really have to override this method, using the TaskMetricGroup is fine.